shared/common-adapters/user-proofs.native.js

```diff
@@ -31,7 +32,7 @@ function MissingProofRow ({missingProof, style}: {missingProof: MissingProof, st
           </Box>
         </Box>
         <Box style={styleStatusIconContainer}>
-          <Icon type={'iconfont-proof-placeholder'} style={{...styleStatusIcon('iconfont-proof-placeholder'), color: missingColor}} />
+          <Icon type={'iconfont-proof-placeholder'} style={{...styleStatusIcon('iconfont-proof-placeholder'), color: globalColors.lightGrey}} />
```

Comment:
Oh no I can't do that because missingColor is used for the text also, and the texts should be black_20